### ISSUE TITLE
Not really headless?

### ISSUE THREAD

**Issue by finkrer** [recodable/react-headless-notifier#2]
First of all, great idea — a headless notification library is exactly what I need.

But it's not really headless, is it? The notification cards are headless, alright. But what about the notification area, the margins between notifications? What about the animations, those are hardcoded.

I think a headless library should limit itself to the logic and leave the styling and animations to the user. As is, this is unusable for me, for example, since we have our own design system and I can't just take someone else's design.

**Comment by excessivecoding** [comment]
I totally agree with you, it's not completely headless. I was planning on working to improve the freedom given to the developer but fell short on time.

The difficulty of headless, and what I was striving for, was to allow flexibility without the complexity of re-implementing everything, library like Headless UI and Radix Primitives choose to give all the freedom to the end developer (which can be great in your case) but has no intermediate level for people choosing to implement simple yet customizable elements.

I think Radix Primitives will suits your need more: https://www.radix-ui.com/docs/primitives/components/toast

PR are always welcome
